connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConnector.java

```diff
@@ -257,6 +261,21 @@ public org.apache.kafka.common.config.Config validate(Map<String, String> props)
                 );
             }
         }
+        boolean offsetSyncsConfigured = configValues.stream()
+                .anyMatch(conf -> conf.name().startsWith(OFFSET_SYNCS_CLIENT_ROLE_PREFIX) || conf.name().startsWith(OFFSET_SYNCS_TOPIC_CONFIG_PREFIX));
+
+        if (!new MirrorSourceConfig(props).emitOffsetSyncEnabled() && offsetSyncsConfigured) {
+            ConfigValue emitOffsetSyncs = configValues.stream().filter(prop -> MirrorConnectorConfig.EMIT_OFFSET_SYNCS_ENABLED.equals(prop.name()))
+                    .findAny()
+                    .orElseGet(() -> {
+                        ConfigValue result = new ConfigValue(MirrorConnectorConfig.EMIT_OFFSET_SYNCS_ENABLED);
```

Comment:
Since we're already importing some constants in `MirrorConnectorConfig`, maybe we should do the same for this one too?